sdk/eventhub/event-hubs/src/util/timerLoop.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * A timer loop is a loop over multiple instances of a promise every specific
+ * time interval. It is different from `setInterval` in that it waits until the
+ * promise is settled before it goes to the next iteration.
+ * */
+export interface TimerLoop {
```

Comment:
feels like we should have a boolean property to know if it's active or not

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/commit/9e1faf1551c63799b5ad35689ebb081699067fa1.